### ISSUE TITLE
Poor performance of marlin_vit_small_ytf

### ISSUE THREAD

**Issue by tvaranka** [ControlNet/MARLIN#7]
Hey, thanks for the great work! I have been using the `marlin_vit_base_ytf` with success when fine-tuning on my custom task. I wanted to try the smaller version `marlin_vit_small_ytf` due to small dataset. However, I find the performance is extremely poor.

With the base model after just 5 epochs I am able to get around 0.2 accuracy, but with the small model after 100 epochs the accuracy is just 0.02.

It almost seems like using a model with random initialization and that the small model hasn't been pre-trained. Would it be possible that you accidentally uploaded the wrong model?

**Comment by tvaranka** [comment]
Nevermind, the model is correct, the issue was on my end.

I was using my local machine for the small model with a small batch size, which seems to have caused the issue.

**Comment by ControlNet** [comment]
Thanks for sharing.
